docs/modules/ROOT/pages/how-to-guides/testing_applications/enable_snyk_check_for_a_product.adoc

```diff
@@ -0,0 +1,46 @@
+= Enabling a Snyk task in Konflux
+
+[abstract]
+The sast-snyk-check task uses the Snyk Code tool to perform static application security testing (SAST). 
+Specifically, the Snyk check scans an application's source code for potential security vulnerabilities, 
+including SQL injection, cross-site scripting (XSS), and code injection attack vulnerabilities.
+
+> NOTE: You can run a Snyk task only if you have a Snyk token stored in a namespace secret. 
+This must also include the name of your secret in the snyk-secret pipeline parameter.
+
+== Enabling Snyk task:
+
+. Register for a Snyk account or log in at https://app.snyk.io/.
+. Get a Snyk token by selecting your Account settings(click on your account name from the bottom left corner) > General > Auth Token section.
+. To enable Snyk code, go to Settings > Snyk Code > Enable Snyk Code > click Enabled.
+. Add this secret to your KONFLUX workspace
+    ** Edit Component to add secret
+        ... Select snyk-secret for the name
+        ... Paste Snyk token into the value text box
+. Update .tekton yaml files to specify the Snyk secret name.
+. Snyk check should now be enabled on the pipeline.
+
+== Obtaining a Red Hat Snyk token using a service account
+[abstract]
+Only for Red Hat Internal users
+
+=== Create organization level Snyk token:
+
+. Register for a Snyk account or log in at https://app.snyk.io/.
+. Select “Group” from the left panel and Choose “Red Hat” from the menu.
+. Select your “Organization” from the left panel.
+. Verify the access on the Organization by clicking on the “Members” button, OrgAdmin is needed in order to create a new service account and a token.
+. Click on “Settings” from the left panel and select “Service accounts”.
+. Give a name to the service account and select the appropriate role from the drop down menu and then click the “Create” button. 
+    ** Recommendation is to keep the token role limited to “Org collaborator” unless some additional rights needed in special cases.
+    ** This token is only visible once, so make sure to copy the token before refresh or close the Web page.
+    ** In case of miss, delete the token and create a new one by following the process described above.
+
+> NOTE: Creating a service account generates an API token that is the only token associated with the service account and takes the place of standard user credentials. Snyk needs authentication in order to initiate Snyk processes.
+This token acts as a replacement for a user authentication, ensure to securely store the token. Do not share it with unintended users and keep rotating the token from time to time to follow security guidelines.
+
+
+== Additional links:
```

Comment:
--> 

[role="_additional-resources"]
== Additional resources

For more information about Snyk, see link:https://snyk.io/product/snyk-code/[the Snyk website].

^^ i know that doesn't render right in GitHub but it will in your ascii editor.